src/internal.c

```diff
@@ -21078,20 +21091,20 @@ int DecodePrivateKey(WOLFSSL *ssl, word16* length)
         #ifdef HAVE_CURVE448
```

Comment:
The Curve448 has key size of 448 but min key size isn't checked.
Similar with Curve25519.

This case statement can be moved below 384.

(Weird rendering of code here.)

Reply:
Fixed